scintilla/lexilla/include/LexicalStyles.iface

```diff
@@ -142,7 +142,8 @@ val SCLEX_HOLLYWOOD=130
 val SCLEX_RAKU=131
 val SCLEX_FSHARP=132
 val SCLEX_JULIA=133
-val SCLEX_GDSCRIPT=134
+val SCLEX_ASCIIDOC=134
+val SCLEX_GDSCRIPT=135
```

Comment:
Why do we need to change this file at all?

Reply:
To match upstream?

Reply:
Looking at history for upstream (yay for it using github not saucefudge) Asciidoc was added before GDscript so the numbers are as per this change.  The question is more why did the GDscript change use the wrong number?

Reply:
My question remains: Why is this file even different from upstream? I imported Lexilla 5.1.4, yet this file wasn't updated during the import.

AFAIU this file should match upstream and we shouldn't have to change it.

Reply:
The change is to match Lexilla upstream file:

https://github.com/ScintillaOrg/lexilla/blob/master/include/LexicalStyles.iface

In Lexilla upstream, GDScripts was added after Asciidoc.

https://github.com/ScintillaOrg/lexilla/commit/a77d85abdc3779a97b9c427399b156f142931543

Reply:
> I imported Lexilla 5.1.4, yet this file wasn't updated during the import.

Bug in update process?  Both Asciidoc and GDscript were in 5.1.4 AFAICT

Reply:
Well, not really a bug. We don't use LexicalStyles.iface for anything, therefore it's not copied on import. I guess I simply copied the whole `lexilla/include` dir when doing the initial lexilla import while the script explicitly only copies `lexilla/include/*.h` (just checked).

So I guess we should decide if we ether drop the file completely or fix the update script to import it correctly. What do you think?